extensions/libraries/redcore/form/fields/rmedia.php

```diff
@@ -63,7 +63,7 @@ protected function getInput()
 			$script = array();
 			$script[] = '	function jInsertFieldValue(value, id) {';
 			$script[] = '		var old_value = document.id(id).value;';
-			$script[] = '		if (old_value != value) {';
+			$script[] = '		if (old_value != value && value) {';
```

Comment:
I think you should swap the order to:
`value && old_value != value`

or even better:
`isset(value) && old_value != value`

or it's still going to show a warning